src/screens/feed/components/FeedPost/FeedPostHeader.tsx

```diff
@@ -9,13 +9,12 @@ import { useUserContext } from 'hooks/context-hooks/useUserContext'
 
 type FeedPostHeaderProps = {
   post: FeedPost
-  showBorder: boolean
+  borderColor: keyof Theme['colors']
```

Comment:
`export type Colors = keyof Theme['colors']`
Please place this type inside theme config file, to avoid writing it from scratch every time :)